src/test/operators/jit_operator/jit_aware_lqp_translator_test.cpp

```diff
@@ -0,0 +1,387 @@
+    ASSERT_NE(jit_read_tuples, nullptr);
+    const auto jit_write_tuples = std::dynamic_pointer_cast<JitWriteTuples>(jit_operators[3]);
+    ASSERT_NE(jit_write_tuples, nullptr);
+
+    const auto output_columns = jit_write_tuples->output_columns();
+    ASSERT_EQ(output_columns.size(), 2u);
+    ASSERT_TRUE(jit_read_tuples->find_input_column(output_columns[0].tuple_value) == ColumnID{2});
+    ASSERT_TRUE(jit_read_tuples->find_input_column(output_columns[1].tuple_value) == ColumnID{0});
+  }
+}
+
+TEST_F(JitAwareLQPTranslatorTest, OutputColumnNamesAndAlias) {
+  const auto jit_operator_wrapper = translate_query("SELECT a, b as b_new FROM table_a WHERE a > 1");
+  const auto jit_operators = jit_operator_wrapper->jit_operators();
+  ASSERT_EQ(jit_operators.size(), 4u);
+
+  const auto jit_write_tuples = std::dynamic_pointer_cast<JitWriteTuples>(jit_operators[3]);
+  ASSERT_NE(jit_write_tuples, nullptr);
+
+  const auto output_columns = jit_write_tuples->output_columns();
+  ASSERT_EQ(output_columns.size(), 2u);
+  ASSERT_EQ(output_columns[0].column_name, "a");
+  ASSERT_EQ(output_columns[1].column_name, "b_new");
+}
+
+TEST_F(JitAwareLQPTranslatorTest, ConsecutivePredicatesGetTransformedToConjunction) {
+  const auto jit_operator_wrapper = translate_query("SELECT a, b, c FROM table_a WHERE a > b AND b > c AND c > a");
+  ASSERT_NE(jit_operator_wrapper, nullptr);
+
+  // Check the type of jit operators in the operator pipeline
+  const auto jit_operators = jit_operator_wrapper->jit_operators();
+  ASSERT_EQ(jit_operator_wrapper->jit_operators().size(), 4u);
+
+  const auto jit_read_tuples = std::dynamic_pointer_cast<JitReadTuples>(jit_operators[0]);
+  const auto jit_compute = std::dynamic_pointer_cast<JitCompute>(jit_operators[1]);
+  const auto jit_filter = std::dynamic_pointer_cast<JitFilter>(jit_operators[2]);
+  const auto jit_write_tuples = std::dynamic_pointer_cast<JitWriteTuples>(jit_operators[3]);
+  ASSERT_NE(jit_read_tuples, nullptr);
+  ASSERT_NE(jit_compute, nullptr);
+  ASSERT_NE(jit_filter, nullptr);
+  ASSERT_NE(jit_write_tuples, nullptr);
+
+  // Check the structure of the computed expression
+  const auto expression = jit_compute->expression();
+  ASSERT_EQ(expression->expression_type(), ExpressionType::And);
+  ASSERT_EQ(expression->left_child()->expression_type(), ExpressionType::And);
+
+  const auto a_gt_b = expression->left_child()->left_child();
+  const auto b_gt_c = expression->left_child()->right_child();
+  const auto c_gt_a = expression->right_child();
+
+  ASSERT_EQ(a_gt_b->expression_type(), ExpressionType::GreaterThan);
+  ASSERT_EQ(a_gt_b->left_child()->expression_type(), ExpressionType::Column);
+  ASSERT_EQ(a_gt_b->right_child()->expression_type(), ExpressionType::Column);
+  ASSERT_TRUE(jit_read_tuples->find_input_column(a_gt_b->left_child()->result()) == ColumnID{0});
+  ASSERT_TRUE(jit_read_tuples->find_input_column(a_gt_b->right_child()->result()) == ColumnID{1});
```

Comment:
@mrzzzrm - This could be shorter and easier, correct?

Reply:
I thought about adding a `==` operator to `JitExpression`. However, this would be the only place where it would be used and in the test case we would still need to manually create the expression to compare to. That is less but noch much less code.
While the current version is quite verbose, it explicitly shows what is being checked instead of calling a "black box" `==` operator. I am happy to change it though if you want.